Resources/Prototypes/_NF/Shipyard/perch.yml

```
@@ -0,0 +1,40 @@
+# Author Info
+# GitHub: lysdiethyl
+# Discord: diethyl
+
+# Shuttle Notes:
+#
+- type: vessel
+  id: Perch
+  parent: BaseVessel
+  name: Perch
+  description: A compact but comfortable shuttle specialized for Vox, featuring salvage capabilities and a relaxation lounge.
+  price: 22000 # ~Sells for about 19500, premium taxation
+  category: Small
+  group: Shipyard
+  shuttlePath: /Maps/_NF/Shuttles/perch.yml
+  guidebookPage: Null
+  class:
+  - Civilian
+  engine:
+  - Plasma
+
+- type: gameMap
+  id: Perch
+  mapName: 'Perch'
+  mapPath: /Maps/_NF/Shuttles/perch.yml
+  minPlayers: 0
+  stations:
+    Perch:
+      stationProto: StandardFrontierVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Perch {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
+            Pilot: [ 0, 0 ]
+            Mercenary: [ 0, 0 ]
```

```suggestion
            ContractorInterview: [ 0, 0 ]
            PilotInterview: [ 0, 0 ]
            MercenaryInterview: [ 0, 0 ]
```
Use new interview jobs.